test/torch/modules/test_torch_distribution_inference.py

```diff
@@ -165,3 +240,64 @@ def test_studentT_likelihood(df: float, loc: float, scale: float):
     assert (
         np.abs(scale_hat - scale) < TOL * scale
     ), f"scale did not match: scale = {scale}, scale_hat = {scale_hat}"
+
+
+@pytest.mark.parametrize("rate", [1.0])
+def test_poisson(rate: float) -> None:
+    """
+    Test to check that maximizing the likelihood recovers the parameters
+    """
+    # generate samples
+    rates = torch.zeros((NUM_SAMPLES,)) + rate
+
+    poisson_distr = Poisson(rate=rates)
+    samples = poisson_distr.sample()
+
+    init_biases = [inv_softplus(rate - START_TOL_MULTIPLE * TOL * rate)]
+
+    (rate_hat,) = maximum_likelihood_estimate_sgd(
+        PoissonOutput(),
+        samples,
+        init_biases=init_biases,
+        num_epochs=20,
+        learning_rate=0.05,
+    )
+
+    assert (
+        np.abs(rate_hat - rate) < TOL * rate
+    ), f"rate did not match: rate = {rate}, rate_hat = {rate_hat}"
+
+
+@pytest.mark.parametrize("total_count_logit", [(2.5, 0.7)])
+def test_neg_binomial(total_count_logit: Tuple[float, float]) -> None:
```

Comment:
Looks like these test parameters are the same as for the mxnet case, but here the parametrization is different